specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -8113,6 +8113,60 @@
           "final-state-via": "azure-async-operation"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/restart": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_Restart",
+        "description": "Restart the dedicated host. The operation will complete successfully once the dedicated host has restarted and is running. For VM health after the restart, please check the Resource Health Center in the Azure Portal. Please refer to https://docs.microsoft.com/en-us/azure/service-health/resource-health-overview for more information.",
```

Comment:
Pls. reword the sentence about resource health as below:
To determine the health of VMs deployed on the dedicated host after the restart check the Resource Health Center in the Azure Portal.  Please refer to https://docs.microsoft.com/en-us/azure/service-health/resource-health-overview for more details.

Reply:
Thanks, wasn't sure exactly how to phrase a message to the user. Will reword to this.